openworld/sdk/core/configuration/client_config.py

```diff
@@ -55,3 +58,7 @@ def auth_config(self) -> AuthConfig:
     @property
     def endpoint(self) -> str:
         return self.__endpoint
+
+    @property
+    def request_timeout(self) -> float:
+        return float(self.__request_timeout)
```

Comment:
Let's do the conversion once we save the value to the object, instead of here (maybe it's not needed, as in 
Python and AFAIK, you should be safe to say that int/int is a float).

Reply:
It is not needed indeed, will remove it.